src/utils.jl

```diff
@@ -245,10 +245,11 @@ end
 # one-liner suggested from ScottPJones
 consumeBOM(buf, pos) = (length(buf) >= 3 && buf[pos] == 0xef && buf[pos + 1] == 0xbb && buf[pos + 2] == 0xbf) ? pos + 3 : pos
 
-if isdefined(Base,:wrap)
-    __wrap(x,pos) = Base.wrap(Array,x,pos)
+if isdefined(Base, :Memory)
```

Comment:
can't you get rid of this entire function and just make it use `view`

Reply:
i want `view(x::Array)` return `x`, and `view(x::Memory)` does not work

Reply:
or do you mean in the function `getbytebuffer` itself?

Reply:
yeah, this function only has one use and now that view isn't a new name from base you can just use it without the check.